### ISSUE TITLE
MathJax mode stuck on loading

### ISSUE THREAD

**Issue by neon64** [shd101wyy/vscode-markdown-preview-enhanced#201]
I used this extension over the competitors because it apparently allows you to use `MathJax` instead of `KaTeX` to get more features. However, when trying to preview a markdown file with `MathJax`, I only get unrendered markdown and a spinning Github logo in the bottom left corner.

Is there any advice on how I should further debug this issue?

**Comment by lingyixia** [comment]
I have met this problem recently too!!!but never before

**Comment by neighthan** [comment]
Just came to post an issue about this. Previously, MathJax support has worked fine. Now I get errors like

```
[Embedded Page] Not allowed to load local resource: file:///C:/extensions/tex2jax.js?V=2.7.1
```

(as well as the same type of error for a few other related files). I'm wondering whether the extension was recently updated to take advantage of the ability to use webpack for your extensions? This is supposed to make things faster/take less memory, but I think something is wrong with the webpack config or how the files are accessed after packing (if I'm right that webpack is involved). [Here] is an issue on the electron repo that I think may be related.

[Here]: https://github.com/electron/electron/issues/5107

**Comment by neighthan** [comment]
#195 looks to be a duplicated (but in Chinese)

**Comment by shd101wyy** [comment]
I will try to fix the mathjax problem this weekend. I am traveling now and I will be back this weekend.

**Comment by neighthan** [comment]
I note your current location is Mars - travel back safely so we can all format our Markdown again!

**Comment by shd101wyy** [comment]
Alright, I just tried to debug the issue and I found that the MathJax only failed on Windows, but not on Linux. So I am wondering are you guys using Windows? Besides, I see MathJax is loading files from some very very weird location like `C:/extensions` and `C:/jax`, so I will need to spend more time debugging why it failed. Thank you for your patience.

**Comment by neighthan** [comment]
I, at least, am on Windows. I was thinking maybe there's some home directory for where it expects the MathJax files that isn't configured properly? This isn't critical for me now, but I'll look forward to a resolution once you have the time.

**Comment by joel-felcana** [comment]
I am on windows too. Noticed that it started to fail just today, but I've used successfully in other documents as recently as the 21th of February.

**Comment by Maxine28Ve** [comment]
I have the same problem and I'm on linux ubuntu 19.10. The math stuff as well as normal text *that wasn't added before opening the .md file* isn't rendered in the markdown preview. There's just a spinning github logo in the bottom left corner of the preview window and that's it
